app/helpers/export_helper.rb

```diff
@@ -93,7 +93,9 @@ def all?
     end
 
     def zip_filename
-      @item.is_a?(User) ? "#{@item.full_name.parameterize}.zip" : "#{@item.name.parameterize}.zip"
+      name_source = @item
+      name_source = @list.first if @list.present? && @list.length == 1
+      name_source.is_a?(User) ? "#{name_source.full_name.parameterize}.zip" : "#{name_source.name.parameterize}.zip"
```

Comment:
```suggestion
      name_source = @list.present? && @list.one? ? @list.first : @item
      base_name = name_source.is_a?(User) ? name_source.full_name : name_source.name
      "#{base_name.parameterize}.zip"
```
I think this is more readable and equivalent?